src/components/Typography/styles.js

```diff
@@ -338,10 +270,17 @@ export const quoteText = {
 }
 
 export const interactionHeadline = {
-  ...editorialHeadline,
+  margin: '0 0 12px 0',
```

Comment:
break the
`UI elements and structured information uses the sans serif cuts. Without margins.`
docu

Reply:
IMO rm and add margin where it's used (title block & teaser) for now